cmds/graph.c

```diff
@@ -835,6 +829,10 @@ int command_graph(int argc, char *argv[], struct opts *opts)
 		synthesize_depth_trigger(opts, &handle, func);
 	}
 
+	/* address length of recorded data might be different from the system */
+	is_lp64 = data_is_lp64(&handle);
+	setup_field_addr(field_table, GRAPH_F_ADDR, is_lp64);
+
```

Comment:
Thanks for doing this.  But I think it'd better passing a pointer to `display_field` to the `print` functions and use the length field.  I know it requires changes in all print functions but it seems the right thing in the long run.

Reply:
Okay. I will think about it. If so, is it okay if I drop this change in this PR so that the first commit can be merged earlier? I think the second commit is not urgent at all so would like to do it maybe next week.